aep/general/0193/aep.md.j2

```diff
@@ -1,5 +1,150 @@
 # Errors
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/48) for more
-information.
+Effective error communication is an important part of designing simple and
+intuitive APIs. Services returning standardized error responses enable API
+clients to construct centralized common error handling logic. This common logic
+simplifies API client applications and eliminates the need for cumbersome
+custom error handling code.
+
+## Guidance
+
+Services **must** clearly distinguish successful responses from error
+responses.
+
+### Structure
+
+An error response **should** contain the following fields:
+
+| Name     | Type    | Required | Description                                                                                                                                                                                                                      |
+| -------- | ------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| type     | string  | Y        | A unique identifier for the type of problem detected.                                                                                                                                                                            |
+| status   | integer |          | The HTTP status code that best describes the type of problem detected.                                                                                                                                                           |
+| title    | string  |          | A human-readable description of the problem. Messages are likely to be logged in plain text and should not include information about a specific occurrence. Information about specific occurrences should be part of `metadata`. |
+| detail   | string  |          | A human-readable explanation specific to this occurrence of the problem.                                                                                                                                                         |
+| instance | string  |          | A unique identifier for the specific occurrence of the problem.                                                                                                                                                                  |
+
+An error response may contain additional fields appropriate for a specific
```

Comment:
`google.rpc.Status` encapsulates this in a details `Any` - which, as an `Any`, (kind of) has a self-describing schema on the wire.  And an API being able to say "here's the schema of `details`" feels cleaner than a flat list of additional properties.  WDYT about prescribing that kind of encapsulation across the board?